client/src/components/graph/graph.tsx

```diff
@@ -961,10 +1053,6 @@ class Graph extends React.Component<GraphProps, GraphState> {
     const { width, height } = this.reglCanvas;
 
     regl?.poll();
-    regl?.clear({
-      depth: 1,
-      color: [1, 1, 1, 1],
-    });
```

Comment:
I believe this is MVP code to paint white background for the image layer, which is no longer needed